integrated_channels/integrated_channel/management/commands/__init__.py

```diff
@@ -0,0 +1,101 @@
+"""
+Enterprise Integrated Channel management commands.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.core.management.base import CommandError
+from django.utils.translation import ugettext as _
+
+from enterprise.models import EnterpriseCustomer
+from integrated_channels.sap_success_factors.models import SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+
+# Import djcelery, or stub it if not available.
+try:
+    from djcelery.celery import task as celery_task
+except ImportError:
+    def celery_task(func):
```

Comment:
@pomegranited I wonder, should we make this `integrated channels` thing a plugin app with it's own dependencies. I mean, we will probably do that at some point, but maybe this point is now.

No need to address this, just thinking out aloud.